teacher.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(name, age, parent_permission)
+    @specialization = specialization
+  end
```

Comment:
In order to call super here without errors, kindly include age, name and parent_permission as the constructor parameters and include the default parameters too.